CalibCalorimetry/HcalAlgos/src/HcalDbHardcode.cc

```diff
@@ -7,6 +7,7 @@
 #include <sstream>
 #include <iostream>
 #include <memory>
+#include <cassert>
```

Comment:
@franzoni , @pohsun , @tlampen , @tocheng , @christopheralanwest can you please sign this change for alca?